app/views/layouts/_header.html.slim

```diff
@@ -0,0 +1,20 @@
+nav class="navbar navbar-expand-lg navbar-light bg-light"
+  div class="container-fluid"
+    == link_to "R-Edited" , root_path , class:"navbar-brand"
```

Comment:
Drošāk izmantot `=`, jo tas tev veikš html escaping - tādā veidā mazāks risks norenderēt kkādu lietotāja html un javascript ievadīto drazu. Paši rails savus būvētos string atzīmē kā html_safe!, tāpēc netiks esceipoti.

Karoč pagaidām - kur vien iespējams, paliec pie `=` instead of `==`

Reply:
sīkāk info šeit https://rdoc.info/gems/slim/frames#output-without-html-escaping

Reply:
a nu sie visi man automatiski uzlikas , kad convertoju no erb uz slim , es parasti izmantoju parasto =